thread/src/components/LoginForm.js

```diff
@@ -19,58 +21,109 @@ class LoginForm extends Component {
-                            <Label>Password</Label>
-                            <Input
-                                secureTextEntry
-                                onChangeText={this.onPasswordChange.bind(this)}
-                                value={this.props.password}
-                            />
-                        </Item>
-                    </Form>
+            <Container style={{ backgroundColor: '#8bc34a' }}>
+
+          {this.logo()}
+                <Content style={styles.viewStyle}>
+
+                        <Form>
+                              <Text style={styles.TextStyle}> Email </Text>
+                                <Item style={styles.inputStyle}>
+                                    <Content>
+                                      <Item>
+                                        <Input style={{color:'#424242'}}
+                                        onChangeText={this.onEmailChange.bind(this)}
+                                        value={this.props.email}
+                                          />
+                                      </Item>
+                                    </Content>
+                                </Item>
+
+                          <Text style={styles.TextStyle}> Password </Text>
+                            <Item style={styles.inputStyle}>
+                                <Content>
+                                  <Item>
+                                    <Input style={{color:'#424242'}}
+                                    secureTextEntry
+                                    onChangeText={this.onPasswordChange.bind(this)}
+                                    value={this.props.password}
+                                      />
+                                  </Item>
+                              </Content>
+                            </Item>
+                        </Form>
+
 
                     {this.renderSignInButton()}
 
-                    <Button bordered full style={{ marginTop: 15 }} onPress={() => Actions.register() }>
+                    <Button rounded warning style={styles.buttonStyle} onPress={() => Actions.register() }>
                         <Text>Register</Text>
                     </Button>
                     <Button transparent full small style={{ marginTop: 15 }} onPress={() => Actions.resetPassword() }>
                         <Text>Forgot password?</Text>
                     </Button>
                 </Content>
             </Container>
+
         );
     }
 }
 
+let styles = StyleSheet.create({
+
+  TextStyle: {
+      color: '#fff',
+      fontSize: 13,
+      alignItems: 'center',
+      justifyContent: 'center',
+      textAlign: 'center',
+      marginTop: 20,
+      fontWeight: 'bold'
+  },
+
+  inputStyle:{
+    width: 340
+  },
+
+  buttonStyle: {
+    marginTop: 25,
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: 330,
+    marginLeft: 20
+  },
+
+  viewStyle: {
+    flex: 1,
```

Comment:
why `flex 1`? For content object from native base ultimately extend using typescript

`/node_modules/react-native/Libraries/Components/ScrollView/ScrollView.js`

This means it inherits its properties as well which includes `flex 1`